src/tennissimulator/match/Match.java

```diff
@@ -0,0 +1,84 @@
+package tennissimulator.match;
+
+import tennissimulator.player.Player;
+
+import java.util.Optional;
+
+public class Match {
+
+    private final Player firstPlayer;
+    private final Player secondPlayer;
+    private ScoreTable scoreTable;
+    public static final int SETS = 3;
+    private Set set;
+    public static final int POINTS_IN_SET_TIEBREAK = 7;
+    public static final int POINTS_IN_MATCH_TIEBREAK = 10;
```

Comment:
private + I would recommend keeping constants together below or above all other fields. It would look cleaner I think

Reply:
fixed